test/createComponent.spec.js

```diff
@@ -26,8 +36,14 @@ describe('createComponent', function () {
         componentDidMount: sinon.match.func,
         componentWillUnmount: sinon.match.func
       });
-    expect(myComponentInstance).to.be.instanceof(MyComponent);
+  });
+
+  it('is a valid React component and a MyComponent\'s instance', function () {
     expect('isReactComponent' in Object.getPrototypeOf(myComponentInstance)).to.be.equal(true);
+    expect(myComponentInstance).to.be.instanceof(MyComponent);
+  });
+
+  it('has the spec\'s functions', function () {
```

Comment:
Oh no, It's fine by me, I was just curious

On Tue, Jul 12, 2016, 16:10 Ricardo Machado <notifications@github.com>
wrote:

> In test/createComponent.spec.js
> <https://github.com/Travix-International/frint/pull/6#discussion_r70445699>
> :
>
> >      expect('isReactComponent' in Object.getPrototypeOf(myComponentInstance)).to.be.equal(true);
> > +    expect(myComponentInstance).to.be.instanceof(MyComponent);
> > +  });
> > +
> > +  it('has the spec\'s functions', function () {
>
> Linting rule: Strings should be either single quoted or, in case we need
> to concatenate with variables, we should use template literals.
>
> But I can improve it to be 'has the functions of the spec'
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Travix-International/frint/pull/6/files/be1ad90c35cf7b537b6e84e28aced553f53c487e#r70445699>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABEf4k8eXqxZKzIcPxmUjnJko0el5cTOks5qU6BygaJpZM4JKVXp>
> .
>